### DISCUSSION TITLE
Swapping bindings for local and global marks don't work?

### DISCUSSION THREAD

**Discussion by rieje** [gdh1995/vimium-c#865] [Q&A]
I have the following bindings:

    # Global marks are more powerful--swap bindings to use lowercase for global and uppercase for local marks
    map X Marks.activateCreateMode swap
    map Y Marks.activateGotoMode swap
    map ' Marks.activate

but `ma` still creates local mark for `a` when it should create global mark and vice versa for `mA`. Seems like it doesn't have any effect. I want this because global marks are more useful than local marks on browers.

--

Unrelated question: is it possible for vomnibar to only show active tabs and ignore inactive tabs? Ideally a binding to show the existing vomnibar for all tabs and another for active tabs only. Very useful when my profile has several hundred tabs.

**Comment by gdh1995** [comment]
You have mapped `X` and `Y` to `Marks.*`. But why did you use `ma` yet? It should have been `Xa`.

What do you mean by `active` and `inactive`? If you have only 1 browser window, then there's only ONE active tab - am I right?

**Reply by rieje** [reply]
> You have mapped `X` and `Y` to `Marks.*`. But why did you use `ma` yet? It should have been `Xa`.

Sorry, I naively thought `map X` and `map Y` are "special" bindings for marks to swap the bindings globally. I guess what I was expecting is a setting to make `m<lowercase_letter>` use global marks and `m<uppercase_letter>` to use local marks since global marks are more useful and typing lowercase letter is easier than uppercase letter. If this is possible (feature request?) then I think it would be much better to stick with `m` for creating marks like I described than to define `X`/`Y` or whatever non-mark related bindings, although it's a decent workaround. Of course it doesn't have to be the default behavior since the current behavior is closer to what it means in vim, but like you suggested in the past global marks are more useful than local marks so IMO should prefer more convenient lowercase `ma` binding (this also means you don't waste `X`/`Y` bindings--sticking with the default `m` is more intuitive. If it's not too difficult to implement this could be an option for user to enable/disable. Not a big deal if you don't agree just want to keep things simple though.

> What do you mean by `active` and `inactive`? If you have only 1 browser window, then there's only ONE active tab - am I right?

Sorry, I meant loaded and unloaded tabs in Firefox. I have profiles with many hundreds of tabs but only have  ~20 active tabs at a time (i.e. tabs I've focused in the running session. the rest are unloaded tabs from previous session on the tab bar).

----------

Another unrelated question regarding vomnibar: other than switching focus to tab when filtering search results and selecting an entry, is it possible to do other things with selected entry? For example, instead of focusing tab, I would like e.g. an option to move selected tab to the current focused tab. This makes it useful to  organize tabs--one can easily move related tabs to be grouped (adjacent) to each other. Another action is e.g. to delete the selected tab (useful when there are duplicate tabs) instead of focusing it then deleting it.
